pkg/kubelet/kubelet_usernamespace_others.go

```diff
@@ -0,0 +1,24 @@
+// +build !linux
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubelet
+
+// Do nothing.
+func (kl *Kubelet) getOwnerIDsFor(path string) (int, int, error) {
+	return -1, -1, nil
+}
```

Comment:
```suggestion
func (kl *Kubelet) getOwnerIDsFor(path string) (uint32, uint32, error) {
	return 4294967295, 4294967295, nil
}
```

Should it really do nothing or return an error?